Source/WebCore/dom/Range.h

```diff
@@ -90,6 +90,7 @@ class Range final : public AbstractRange, public CanMakeSingleThreadWeakPtr<Rang
 
     Ref<DOMRectList> getClientRects() const;
     Ref<DOMRect> getBoundingClientRect() const;
+    static Ref<DOMRect> getBoundingClientRect(const SimpleRange&);
```

Comment:
Could eventually move this somewhere else since it's not part of the `Range` class any more but it's fine for now. Also could name it using WebKit coding style without the "get" since it's not directly web-exposed.

Reply:
I dropped the "get".